projects/angular/src/emphasis/label/label.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2016-2025 Broadcom. All Rights Reserved.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Component, Input } from '@angular/core';
+
+import { ClrIconModule } from '../../icon';
+import { ClrBadge } from '../badge';
+
+export enum ClrLabelColors {
+  Empty = '',
```

Comment:
Empty doesn't sound as a state for color, maybe none or default fits better.
```suggestion
  None = '',
```

Reply:
Corrected in commit 1b671e5